packages/react-native-flyy/README.md

```diff
@@ -0,0 +1,29 @@
+# @config-plugins/react-native-flyy
+
+Config plugin to auto configure Flyy SDK on prebuild
+
+# API documentation
+
+- [Documentation for the master branch](https://github.com/expo/expo/blob/master/docs/pages/versions/unversioned/sdk/@config-plugins/react-native-flyy.md)
+- [Documentation for the latest stable release](https://docs.expo.io/versions/latest/sdk/@config-plugins/react-native-flyy/)
+
+# Installation in managed Expo projects
+
+For managed [managed](https://docs.expo.io/versions/latest/introduction/managed-vs-bare/) Expo projects, please follow the installation instructions in the [API documentation for the latest stable release](#api-documentation). If you follow the link and there is no documentation available then this library is not yet usable within managed projects &mdash; it is likely to be included in an upcoming Expo SDK release.
+
+# Installation in bare React Native projects
+
+For bare React Native projects, you must ensure that you have [installed and configured the `react-native-unimodules` package](https://github.com/expo/expo/tree/master/packages/react-native-unimodules) before continuing.
```

Comment:
react-native-unimodules is deprecated since SDK 43

Reply:
react-native-unimodules instruction removed from README